### PR TITLE
Update zick.json

### PR DESCRIPTION
**Pull request by LucasZick**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] You're not using Vercel or Netlify.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.

### CODE REVIEW

**Comment by LucasZick** [comment]
Honestly, I'm not sure on how to work with the zoho mail + domain, I just added what it asked me to, if its wrong can you please give me the directions for it to work?

File: domains/zick.json [hunk 1]
```diff
@@ -4,6 +4,11 @@
         "email": "lucasalexandrezick@gmail.com"
     },
     "record": {
-        "A": ["144.22.193.227"]
+        "A": ["144.22.193.227"],
+        "MX": "mx.zoho.com",
+        "MX": "mx2.zoho.com",
+        "MX": "mx3.zoho.com",
+        "TXT": "v=spf1 include:zoho.com ~all",
+        "TXT": "v=DKIM1; k=rsa; p=MIGfMA0GCSqGSIb3DQEBAQUAA4GNADCBiQKBgQCdZp0j5EFJS+OJzSad09+rt4K73CiPfeNahs2qXKBdaRLSrELeRbh4m8dccFi81fAjtPXpU5EonVVj4fssytg06MJCJWbdb02NqwZGS3tm29maZV2UFBb0Fqj4uqNRy1LKynl37jA8XkshE0xyteyFvX3d9Ibf4n7W1VlDhF1HpwIDAQAB"
```
Comment (DEV-DIBSTER) [inline domains/zick.json:12]: TXT records should be combined into array. Same with MX records. They should be similar to your A records.

**Review by DEV-DIBSTER — DISMISSED** [review]

File: domains/zick.json (hunk 1 above)
Comment (LucasZick) [reply domains/zick.json]: Thank you! Solved it!

**Comment by LucasZick** [comment]
Solved the requested changes, how do I proceed?

**Review by VaibhavSys — APPROVED** [review]

**Pull request merged**